conductor/submitter_maya.py

```diff
@@ -295,6 +309,13 @@ def runPreSubmission(self):
         in the returned dictionary (so that we don't need to collect them again).
         '''
 
+        # Check if scene has unsaved changes and ask user if they'd like to
+        # save their scene before continuing with submission
+        if not self.checkSaveBeforeSubmission():
+            raise exceptions.UserCanceledError()
+        else:
```

Comment:
Nit: `else` condition is not needed if the result of the `if` branch is an exception.

Reply:
JESUS FUCKING CHRIST. Get out of my head!  
I'm curious, is there any actual literature (code styling guides, books, etc)  that illustrate this particular point (and the other nit earlier)?  As much as I would love for people to code to in accordance to my own preferences/style, these suggestions don't usually fly when imposing them on other developers.   (everyone's mind works differently, pride, etc).  Unless there's an  external/credible source of "authority" that can be pointed to, I find that it's hard to coerce developers into changing their ways (understandably).  This particular nit reminds me of @jamesm-cio strong preference for a function to have only one exit point, where I don't necessarily agree.   Though it sounds simple (and simple is good, yah?), sometimes having multiple exit points (particularly if they are very near the entrance point) can make for easier reading with more intuitive logic (at least in my own head).   But beauty is always in the eye of the beholder :)  

What's a highly-opinionated,  language enthusiast to do?!

Reply:
Why not codify a code style guide for the team to adhere to, socialize it, then set up all the code linters to apply it?

Reply:
Some companies release their code style guides to the public. We could look for one that we can all stand behind and implement it. Let's discuss that as a team next week.

Reply:
I generally strive to adhere to one exit point per function - but as @lawschlosser mentioned, it is easier said than done, particularly when in legacy code.   In this case the logic flow during pre-submission includes using a custom exception to bail out (cancel) the user operation - I don't mind the design all too much, so I went along with it.